cpmpy/solvers/solver_interface.py

```diff
@@ -76,6 +76,7 @@ def __init__(self, name="dummy", cpm_model=None, subsolver=None):
         # initialise variable handling
         self.user_vars = set()  # variables in the original (non-transformed) model
```

Comment:
Iguess this is a use case for cpm_set? a hash collision on the Var would not resolve

Reply:
yes.. I guess we will have to do some benchmarking to see if we really want to make this hash-collision proof